sklearn/feature_extraction/image.py

```diff
@@ -372,6 +373,8 @@ def extract_patches_2d(image, patch_size, *, max_patches=None, random_state=None
         The maximum number of patches to extract. If `max_patches` is a float
         between 0 and 1, it is taken to be a proportion of the total number
         of patches.
+        If max_patches=None then the total number of patches will be extracted.
+
```

Comment:
We already merged a previous PR that was doing this change in `main`. You can remove this line.